books/free-programming-books-he.md

```diff
@@ -0,0 +1,36 @@
+### Index
+
+* [ללא תלות בשפה](#ללא-תלות-בשפה)
+  * [מערכות הפעלה](#מערכות-הפעלה)
+  * [רשתות](#רשתות)
+* [Assembly](#assembly)
+* [C#](#c-sharp)
+* [Python](#python)
+
+
+### ללא תלות בשפה
+
+#### מערכות הפעלה
+
+* [מערכות הפעלה](https://data.cyber.org.il/os/os_book.pdf) – ברק גונן, המרכז לחינוך סייבר (PDF)
+
+
+#### רשתות
+
+* [רשתות מחשבים](https://data.cyber.org.il/networks/networks.pdf) – עומר רוזנבוים, ברק גונן, שלומי הוד, המרכז לחינוך  (PDF)סייבר
```

Comment:
@tomer, should this be reword as like in other `ciber` resources? I found a bit strange this glyphs after the (PDF) tag

`עומר רוזנבוים, ברק גונן, שלומי הוד, המרכז לחינוך (PDF)`**`סייבר`**

`המרכז לחינוך` -> center of education
`סייבר` -> ciber

Reply:
You right. It caused because the online editor doesn't respect RTL content.